pkg/daemon/daemon.go

```diff
@@ -247,6 +250,13 @@ func cmdRun(p *ptpProcess) {
 	//
 	// don't discard process stderr output
 	//
+	c, err := net.Dial("unix", "/tmp/metrics.sock")
+	if err != nil {
+		glog.Errorf("Dial error", err)
+		return
+	}
+	defer c.Close()
```

Comment:
better to put this in the defer already exist above

Reply:
done